<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::SettingsDialog</class>
 <widget class="QDialog" name="ns::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>715</width>
    <height>535</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>733</width>
    <height>553</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetOuter {
	border: 1px solid #4D5F6B;
	border-radius: 4px;
}

#widgetInner {
	border: 1px solid #F7FCFF;
	border-radius: 4px;
}

/* css of caption */
#frameCaption {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #0078c7, stop: 1.0 #016aae);
	border-top: 1px solid #58A8DB;
	border-right: 1px solid #58A8DB;
	border-left: 1px solid #58A8DB;
	border-bottom: 1px solid #58A8DB;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
}

#frameCaption #labelTitle {
	font-size: 13px;
	font-weight: bold;
	color: #ffffff;
	padding-left: 6px;
}

#frameCaption #sysPanelContainer {
	border: 0px;
}

#frameCaption #sysPanelContainer #sysPanel {
	border: 0px;
}

#frameCaption #pushButtonClose {
	border-image: url(:/images/ui/caption_close.png);
}

#frameCaption #pushButtonClose:hover {
	border-image: url(:/images/ui/caption_close_hover.png);
}

#frameCaption #pushButtonClose:pressed {
	border-image: url(:/images/ui/caption_close_pressed.png);
}

/* end of css caption */

#widgetButtonContainer {
	background: #E1F1FE;
	border-top: 1px solid #FCFDFF;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
}

#widgetButtonContainer #pushButtonOK {
	border-image: url(:/images/ui/setting_ok.png) 0 3 0 3;
	border-top: 0 transparent;
	border-right: 3 transparent;
	border-bottom: 0 transparent;
	border-left: 3 transparent;
}

#widgetButtonContainer #pushButtonOK:hover {
	border-image: url(:/images/ui/setting_ok_hover.png) 0 3 0 3;
}

#widgetButtonContainer #pushButtonOK:pressed {
	border-image: url(:/images/ui/setting_ok_pressed.png) 0 3 0 3;
}

#widgetButtonContainer #pushButtonCancel {
	border-image: url(:/images/ui/setting_cancel.png) 0 3 0 3;
	border-top: 0 transparent;
	border-right: 3 transparent;
	border-bottom: 0 transparent;
	border-left: 3 transparent;
}

#widgetButtonContainer #pushButtonCancel:hover {
	border-image: url(:/images/ui/setting_cancel_hover.png) 0 3 0 3;
}

#widgetButtonContainer #pushButtonCancel:pressed {
	border-image: url(:/images/ui/setting_cancel_pressed.png) 0 3 0 3;
}

#indexTree {
	border: 0px;
	background: #E7F4FD;
	show-decoration-selected: 1;
	padding: 8px 3px 8px 3px;
}

#indexTree::item {
	margin: 1px;
	border-top-color: transparent;
	border-bottom-color: transparent;
	height: 23px;
}

#indexTree::item:selected {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #3593e2, stop: 1.0 #2482d1);
	color: #ffffff;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="window" native="true">
     <property name="minimumSize">
      <size>
       <width>715</width>
       <height>535</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>715</width>
       <height>535</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>35</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetOuter" native="true">
        <property name="minimumSize">
         <size>
          <width>643</width>
          <height>463</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>643</width>
          <height>463</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="QWidget" name="widgetInner" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>461</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>461</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frameCaption">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelTitle">
                 <property name="text">
                  <string>Settings</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>68</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QFrame" name="sysPanelContainer">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="sysPanel" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="pushButtonClose">
                       <property name="minimumSize">
                        <size>
                         <width>35</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>35</width>
                         <height>36</height>
                        </size>
                       </property>
                       <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="autoDefault">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>9</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widgetBottom" native="true">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widgetMain" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QTreeWidget" name="indexTree">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                      <horstretch>1</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>188</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>188</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="indentation">
                     <number>0</number>
                    </property>
                    <attribute name="headerVisible">
                     <bool>false</bool>
                    </attribute>
                    <attribute name="headerDefaultSectionSize">
                     <number>0</number>
                    </attribute>
                    <column>
                     <property name="text">
                      <string>1</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widgetPageContainer" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>453</width>
                      <height>397</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>453</width>
                      <height>397</height>
                     </size>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QStackedWidget" name="pages">
                       <property name="currentIndex">
                        <number>4</number>
                       </property>
                       <widget class="ns::SettingsGeneralPage" name="generalPage"/>
                       <widget class="ns::SettingsLanguagePage" name="languagePage"/>
                       <widget class="ns::SettingsHintPage" name="hintPage"/>
                       <widget class="ns::SettingsCallinPage" name="callinPage"/>
                       <widget class="ns::SettingsBlacklistPage" name="blacklistPage"/>
                       <widget class="ns::SettingsUpdatePage" name="updatePage"/>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widgetButtonContainer" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>37</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>37</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="topMargin">
                   <number>9</number>
                  </property>
                  <property name="bottomMargin">
                   <number>9</number>
                  </property>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>470</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonOK">
                    <property name="minimumSize">
                     <size>
                      <width>64</width>
                      <height>21</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>64</width>
                      <height>21</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string>OK</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonCancel">
                    <property name="minimumSize">
                     <size>
                      <width>64</width>
                      <height>21</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>64</width>
                      <height>21</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string>Cancel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ns::SettingsGeneralPage</class>
   <extends>QWidget</extends>
   <header>settingsgeneralpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ns::SettingsLanguagePage</class>
   <extends>QWidget</extends>
   <header>settingslanguagepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ns::SettingsHintPage</class>
   <extends>QWidget</extends>
   <header>settingshintpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ns::SettingsCallinPage</class>
   <extends>QWidget</extends>
   <header>settingscallinpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ns::SettingsBlacklistPage</class>
   <extends>QWidget</extends>
   <header>settingsblacklistpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ns::SettingsUpdatePage</class>
   <extends>QWidget</extends>
   <header>settingsupdatepage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>indexTree</tabstop>
 </tabstops>
 <resources>
  <include location="../translator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>ns::SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ns::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ns::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
